src/core/positioning/gnsspositioninformation.h

```diff
@@ -109,6 +110,15 @@ class GnssPositionInformation
 
     Q_ENUM( FixStatus )
 
+    enum AccuracyQuality
+    {
+      AccuracyBad,
+      AccuracyOk,
+      AccuracyExcellent
+    };
```

Comment:
Let's an an undetermined state. Bad by default is, really bad :)

```suggestion
    enum AccuracyQuality
    {
      AccuracyUndetermined,
      AccuracyBad,
      AccuracyOk,
      AccuracyExcellent
    };
```